cirq/ops/fsim_gate.py

```diff
@@ -66,6 +66,9 @@ def _is_parameterized_(self):
         return cirq.is_parameterized(self.theta) or cirq.is_parameterized(
             self.phi)
 
+    def _has_unitary_(self):
+        return not self._is_parameterized_()
+
     def _unitary_(self) -> Optional[np.ndarray]:
         if cirq.is_parameterized(self):
```

Comment:
Nit: Inconsistent with other places.

Reply:
In what way?

Reply:
Often `self._is_parameterized_()` is used instead.